7-layouts/style.css

```
@@ -0,0 +1,89 @@
+body {
+    color: whitesmoke;
+    margin: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+footer {
+    background-color: rgb(39, 39, 39);
```

```suggestion
    background-color: #272727;
```
If you don't use rgba + the opacity value, use hexadecimal for colors.